### ISSUE TITLE
Agent cards can't be altered no more when the original owner is cloned.

### ISSUE THREAD

**Issue by Allspice** [vgstation-coders/vgstation13#17852]
I'm guessing it goes off of body, like wizard spells. It should go off of DNA instead if anything (and I THINK cloning doesn't alter DNA at the moment).

**Comment by Pathid** [comment]
Apparently it is locked to your mob?
https://github.com/vgstation-coders/vgstation13/blob/f90af69bcea7d6f47f49d33fe37872a11db7ad10/code/game/objects/items/weapons/cards_ids.dm#L418-L422

You're right, it should probably be ``user.dna.unique_enzymes`` instead of user.

**Comment by Blithering** [comment]
does mind carry over to golems and diona? If so that would be an issue. Could also just have it only operable by syndicates, ops and raiders unless that's how it currently works

**Comment by Aurx** [comment]
Making it antag-use-only would let it antag-test, albeit only against scrubs.

**Comment by Blithering** [comment]
It's not like they can force you to change the ID, you can just say 'nope, tried it, doesn't work'

**Comment by Aurx** [comment]
Assumes the tator knows it's tator only and they're being tested.  There's like, a dozen ways to frame it as "dude here use this".

**Comment by Blithering** [comment]
True, but then you could say the same for asking suspected cultists what a rune says. maybe have some kind of confirmation box when altering the ID that warns them not to do so in public with a cancel button?

**Comment by Pathid** [comment]
>does mind carry over to golems and diona?

No. That's the point of mind. It only tracks with your IC "soul" and gets left behind when you become a golem or diona.

**Comment by Blithering** [comment]
Ah fair enough then. That seems a lot more elegant than my rambling solution
